src/lib/dot-matrix/custom-hooks/useChartContainerWidth.ts

```diff
@@ -29,8 +25,10 @@ const useChartContainerWidth = (
 
   const updateContainerWidth = (): void => {
     let widthValue;
-    if (typeof window !== undefined) widthValue = findContainerWidth(id);
-    if (widthValue) setWidth(widthValue);
+    if (typeof window !== undefined) {
```

Comment:
```suggestion
    if (typeof window !== 'undefined') {
```

typeof returns string